docs/maintaining/tasks.md

```diff
@@ -18,8 +18,9 @@ To build new images and publish them to the Docker Hub registry, do the followin
 
 ## Updating the Ubuntu Base Image
 
-`minimal-notebook` is based on the Latest LTS Ubuntu docker image.
-Other images are directly or indirectly inherited from `minimal-notebook`.
+`base-notebook` is based on the LTS Ubuntu docker image.
+We wait for the first point release of the new LTS Ubuntu before updating the version.
```

Comment:
@consideRatio I agree with you, that's why I've already changed the docs to represent how we do it :)

Reply:
Also, the official name for Ubuntu is `point release`, so I've decided to use it here.
https://wiki.ubuntu.com/PointReleaseProcess